test/setup/main.tf

```diff
@@ -145,16 +145,50 @@ module "vpc" {
   network_name    = "eab-vpc-${each.key}"
   shared_vpc_host = true
 
+  egress_rules = [
+    {
+      name     = "allow-private-google-access"
+      priority = 200
+      destination_ranges = [
+        "34.126.0.0/18",
+        "199.36.153.8/30",
+      ]
+      allow = [
+        {
+          protocol = "tcp"
+          ports    = ["443"]
```

Comment:
Is there public documentation for allowing just port 443?  I found docs for the range and TCP (HTTP, HTTPS, HTTPS/2), but not a specific (HTTPS) port number.

- https://cloud.google.com/vpc/docs/configure-private-google-access
- https://cloud.google.com/vpc-service-controls/docs/set-up-private-connectivity

Reply:
There is a doc saying that all traffic to `restricted.googleapis.com` is to port 443: https://cloud.google.com/vpc-service-controls/docs/set-up-private-connectivity#configure-firewall (but I have not found the same information for `private.googleapis.com`)

In the `terraform-example-foundation` the firewall rule for the Private Google Access network is using only port 443 also: https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks-dual-svpc/modules/base_shared_vpc/firewall.tf#L58

Reply:
Thanks! I think this is sufficient for now.